azure-container-apps/README.md

```diff
@@ -181,17 +181,17 @@ If you're provisioning more than 1,000 users, update the resources assigned to [
 
 ### How to update the **scimsession** secret
 
-After you download a new `scimsession` file, follow the steps below to replace the secret in your Container App.
+To use a new `scimsession` credentials file for your SCIM Bridge, follow replace the secret in your Container App:
 
-Replace your <code>scimsession</code> secret using the Azure Portal
+Replace your `scimsession` secret using the Azure Portal
```

Comment:
Looks like this section is missing the `<details>` element to make it expandable. Your PR didn't do that, though, so it must have gotten removed at some point. Can you add it back like the section in line 197 so they're both collapsible?

Reply:
You can blame yourself and @black-bryan for that 😜: 

https://github.com/1Password/scim-examples/blame/9f8ec7d558d26a6bacb69380f869985b0245168b/azure-container-apps/README.md#L186

Reply:
1 of 2:
```suggestion
<details>
<summary>Replace your `scimsession` secret using the Azure Portal</summary>
```